great_expectations/checkpoint/checkpoint.py

```diff
@@ -407,7 +407,7 @@ def _run_validation(
         ) as e:
             raise ge_exceptions.CheckpointError(
                 f"Exception occurred while running validation[{idx}] of Checkpoint '{self.name}': {e.message}."
-            )
+            ) from e
```

Comment:
what exactly does this do?

Reply:
If you catch an exception and raise a different on the original backtrace of the first exception gets lost. This can sometimes be a problem when root causing the error. the `from previous_exception` notation makes the backtrace look something like:
```
<Original exception backtrace>
when handling that exception another exception occurred
<backtrace thrown here>
```

Reply:
nevermind did some of my own research 😄